### DISCUSSION TITLE
Is there a new and easy way to show the axact day from 00:00 to 00:00 without using parameter offset

### DISCUSSION THREAD

**Discussion by thoath2** [dbuezas/lovelace-plotly-graph-card#500] [Q&A] [answered]
Hello,
at the moment I'm using the following code to represent exactly one calendar day (from 00:00 to 00:00). 
```
hours_to_show: 24
time_offset: |-
  $fn ({ getFromConfig }) => {
    const ms_to_show = getFromConfig('hours_to_show') * 60 * 60 * 1000;
    const now_utc = Date.now() - new Date().getTimezoneOffset() * 60 * 1000;
    return ms_to_show - now_utc % ms_to_show + 'ms'
  }
```
But I find this solution too complicated. And I'm not sure if it has any unpleasant side effects.
I read somewhere that there will soon be a simple configuration solution. Can someone tell me if this simple solution already exists and if so, how do I configure it?
Thanks

**Comment by dbuezas** [comment, marked as answer]
https://github.com/dbuezas/lovelace-plotly-graph-card?tab=readme-ov-file#dynamic-relative-time

Did you see that

**Reply by thoath2** [reply]
Oh, thank you so much. I read through the Github page so many times and couldn't find the setting options. In German we have an expression: "I had tomatoes in front of my eyes"

**Reply by dbuezas** [reply]
Kein Problem!

**Comment by thoath2** [comment]
Thanks, i close the ticket now
